converters/toZigbee.js

```diff
@@ -410,7 +418,10 @@ const converters = {
                 } else if (message.hasOwnProperty('brightness_percent')) {
                     brightness = Math.round(Number(message.brightness_percent) * 2.55).toString();
                 }
-
+                if (brightness === 255) {
```

Comment:
Can be simplified to `brightness = Math.min(254, brightness)`

Reply:
Yes, you can change to most simple code.